i3pystatus/now_playing.py

```diff
@@ -100,14 +100,20 @@ def run(self):
         try:
             player = self.get_player()
             properties = dbus.Interface(player, "org.freedesktop.DBus.Properties")
-            get_prop = functools.partial(properties.Get, "org.mpris.MediaPlayer2.Player")
+
+            def get_prop(name, default=None):
+                try:
+                    return properties.Get("org.mpris.MediaPlayer2.Player", name)
+                except:
```

Comment:
Can you make this `except` clause less broad? It probably raises AttributeError...